sdk/storage/perf-tests/storage-blob/tsconfig.json

```diff
@@ -1,7 +1,6 @@
 {
   "extends": "../../../../tsconfig.package",
   "compilerOptions": {
-    "target": "es5",
```

Comment:
es5 target is hitting me with this
![image](https://user-images.githubusercontent.com/10452642/127281618-316163aa-fdc4-471b-8f70-0563ab95f529.png)

I figured.. there's no harm in upgrading.(by removing target es5)